atoman/gui/dialogs/infoDialogs.py

```diff
@@ -679,8 +682,10 @@ def __init__(self, pipelinePage, atomIndex, scalarsDict, vectorsDict, filterList
         self.filterList = filterList
         self.neighbourInfoWindow = None
         
-        lattice = self.pipelinePage.inputState
+        self.PBC = [1,1,1]
```

Comment:
Actually I don't think I can edit the pull request as it is your branch - you can just commit to your branch and push and it will update the pull request. No worries though, I can just accept and change it later...